enhancements/assets-generation/cloud-foundry-application-discovery/README.md

```diff
@@ -119,44 +119,42 @@ See [metadata specification](#metadata-specification). |
 | **no-route** | Y | Routes | Processes will have no route information in the canonical form manifest. See [process specification](#process-specification). |
 | **processes** | Y | Processes | See [process specification](#process-specification) |
 | **random-route** | Y | Routes | See [route specification](#route-specification). |
-| **default-route** | Y |  | The default behavior for CF Applications is to define a route based on the name of the application and its domain ([https://docs.cloudfoundry.org/devguide/deploy-apps/deploy-app.html\#default-push](https://docs.cloudfoundry.org/devguide/deploy-apps/deploy-app.html#default-push)).
-Example:
-https://docs.cloudfoundry.org/devguide/multiple-processes.html |
+| **default-route** | Y |  | The [default behavior](https://docs.cloudfoundry.org/devguide/deploy-apps/deploy-app.html#default-push) for CF Applications is to define a route based on the name of the application and its domain. See an [example](https://docs.cloudfoundry.org/devguide/multiple-processes.html). |
 | **routes** | Y | Routes | See [route specification](#route-specification). |
 | **services** | Y | Services | See [service specification](#service-specification). |
 | **sidecars** | Y | Sidecars | See [sidecar specification](#sidecar-specification). |
 | **metadata** | Y | Metadata | See [metadata specification](#metadata-specification). |
 | **buildpack** | N |  | Already deprecated in CF. See buildpacks. |
-| **timeout** | Y | StartupTimeout |  |
-| **instances** | Y | Replicas |  |
-| **stack** | Y | Stack |  |
+| **timeout** | Y | StartupTimeout | Maximum time allowed for an application to respond to readiness or health checks during startup.If the application does not respond within this time, the platform will mark the deployment as failed. |
+| **instances** | Y | Instances | Number of CF application instances |
```

Comment:
I'll change the reference in the go structure to `instances`

Reply:
replicas in back :) let me know what you think :)

Reply:
I was ok with `instances`, which is what is called in CF 😄 . Conceptually is the same, but I was convinced by `instances` when I spotted it here when I realized that we aim to stick as much as possible to the CF naming convention whenever possible (except, for instance `timeout` -> `startTimeout` because it makes more sense to me and it represents what the field is for).

But I won't argue if you'd rather stick to `replicas`... 😜